packages/config/src/options.ts

```diff
@@ -419,6 +419,11 @@ const resolvedOptions: Array<ResolvedConfigOption> = [
     canUpdateDuringTestTime: false,
     requireRestartOnChange: 'server',
   },
+  {
+    name: 'specPattern',
```

Comment:
This is a good catch! We seem to also declare the default values in other cases.  Looks like we can use the same `options.testingType` ternary that's being used for `viewportWidth` etc to populate the defaults that are declared in this file already in the `defaultSpecPattern` object. What do you think?

Reply:
I added that but then I just removed it again... I'm confused about what `defaultValue` does here. It looks like it was messing up some of the validation in the tests when I had it in there 😕

Reply:
Let's make an issue and add a comment for this then, totally worth investigate.

Reply:
Added an issue and a comment here